lib/launcher.rb

```diff
@@ -87,7 +96,7 @@ def https_app(https_port, plugins = https_plugins)
 
       {
         :app => app,
-        :server => :webrick,
+        :server => Proxy::SETTINGS.http_server_type || :webrick,
```

Comment:
This does not correspondend with the line above `:server => Proxy::SETTINGS.http_server_type.to_sym,` so either one or another. The settings stack handles the defaults, so I think the version from 58 is better.